irm/noxfile.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import os
+
+import nox
+
+
+LOCAL_DEPS = (os.path.join("..", "api_core"), os.path.join("..", "core"))
+
+@nox.session(python="3.7")
+def lint(session):
+    """Run linters.
+
+    Returns a failure if the linters find linting errors or sufficiently
+    serious code quality issues.
+    """
+    session.install("flake8", "black", *LOCAL_DEPS)
+    session.run(
+        "black",
+        "--check",
+        "google",
+        "tests",
+        "docs",
+    )
+    session.run("flake8", "google", "tests")
+
+
+@nox.session(python="3.6")
```

Comment:
It seems all of the other sessions are running the latest python. Is there a reason black isn't?
```suggestion
@nox.session(python="3.7")
```

Reply:
Hmm, looks like it is codegen which is generating the "3.6".